### ISSUE TITLE
v12 cluster upgrade: custom logo not pushed on other nodes

### ISSUE THREAD

**Issue by lzammit** [inverse-inc/packetfence#7243]
Situation:

Upgrade a server prior v12 with a custom logo on the captive portal under html/common/.

Once you bootstrap your first pf1 node with the export.tgz, everything works but the html/common/custom-logo.png is not pushed into the pf2 and pf3 httpd.portal docker image.

How do we handle custom logo upload in v12?

**Comment by julsemaan** [comment]
@fdurand said he will take a look at this for 12.0 specifically

For 12.1, we'll target to have an upload of the logo in the admin interface

**Comment by nqb** [comment]
On my side, I tried this on a `12.0.0` setup:

```
cp /usr/local/pf/html/common/log.png /usr/local/pf/html/captive-portal/profile-templates/default/logo.png
```
I can see that the file appears inside httpd.portal Docker image
After I edited my default connection profile like this:

```
[default]
logo=/profile-templates/default/logo.png
```

Then I tried to reach captive portal and got following error:
```
GET https://portal.example.lan/profile-templates/default/logo.png
501 Not Implemented
VersionHTTP/1.1
```
Are we sure we can reach directly files under `profile-templates` ?

**Comment by julsemaan** [comment]
I think it used to be true but isn't anymore with the httpd.dispatcher serving the static assets

**Comment by nqb** [comment]
In order to apply fix provided by #7261 and #7288 on a 12.0.0 setup, you shoud follow this process:
1. [Upgrade to latest patch version](https://www.packetfence.org/doc/PacketFence_Upgrade_Guide.html#_upgrade_to_a_patch_version_x_y_patch)
   * In a cluster environment, perform this operation on all cluster members
1. Once services have been restarted:
   * run `/usr/local/pf/addons/upgrade/move-logos-to-profile-templates.pl`
      * In a cluster environment, perform this operation on a cluster member where your logos exist (look into `/usr/local/pf/html/common`)
    * run  `/usr/local/pf/bin/pfcmd configreload hard` where you execute `move-logos-to-profile-templates.pl` script
    * In a cluster environment, run following commands on cluster member where you execute `move-logos-to-profile-templates.pl` script:
```
    /usr/local/pf/bin/cluster/sync --as-master
```

Your logos should appear on captive portal and also when using "Preview" feature on webadmin.

**Note**: if you previously synchronized files under `/usr/local/pf/html/common`, you can ignore errors related to these files which appear when running `/usr/local/pf/bin/cluster/sync --as-master`
